src/config/build_config.py

```diff
@@ -1,8 +1,8 @@
 # Please only set it to True if you are preparing an EI related PR
 # Do remember to revert it back to False before merging any PR (including EI dedicated PR)
-ENABLE_EI_MODE = False
+ENABLE_EI_MODE = True
 # Frameworks for which you want to disable both builds and tests
-DISABLE_FRAMEWORK_TESTS = []
+DISABLE_FRAMEWORK_TESTS = ["mxnet","tensorflow"]
```

Comment:
If changes are being made to TensorFlow and MXNet dockerfiles, please enable their builds/tests here.

Reply:
This is for temporary as we are trying to debug the PT failure issue currently.